src/slot.rs

```diff
@@ -300,31 +322,41 @@ impl<Conn: Connection> Slot<Conn> {
         };
 
         #[cfg(feature = "probes")]
-        let id = usdt::UniqueId::new();
-        #[cfg(feature = "probes")]
-        probes::recycle__start!(|| (&id, backend.address.to_string()));
```

Comment:
This is an example, where we are using the "ID of the slot" rather than "an arbitrary ID" to help identify what is happening.

This will, hopefully, make it easier to correlate "probe creation -> checkout -> recycle" info